src/components/POLine/POLineDetails/FieldOrderFormat.js

```diff
@@ -37,8 +37,8 @@ class FieldOrderFormat extends Component {
   onChangeSelect = (value) => {
     const { dispatch, change, store, vendors, orderVendorId } = this.props;
 
-    dispatch(change('cost.quantity_physical', ''));
-    dispatch(change('cost.quantity_electronic', ''));
+    dispatch(change('cost.quantityPhysical', ''));
+    dispatch(change('cost.quantityElectronic', ''));
```

Comment:
Did you miss activation_due below (line 45)?

Reply:
and line 49?

Reply:
Missed, thanks!